### ISSUE TITLE
Confirm Multisign with Differents Wallets 

### ISSUE THREAD

**Issue by CodersBrothers** [bitpay/copay#4713]
If I have an Armory, a Copay and a other multisign wallet, Is possible complete a multisign wallet and later a multisign transaction?

If I understand right, when I want (with copay) create a multisignature wallet, copay creates a "secret" (random and unique hash like an ID) for link public keys of each party. But this only can be with other copay's, because only copay is waiting for this. Instear other custom copies of copay multisignture wallet will be inposible, only with the same server (database). The feature requires a central server (same server for all) to manage this as easy way, Right?

**Comment by dabura667** [comment]
I am sorry, I can't understand what your issue is.

Could you rephrase the problem you are experiencing?

**Comment by CodersBrothers** [comment]
the question is: It is possible sign a multisign transaction with differents copay clients if these clients are using differents backend servers?

**Comment by matiu** [comment]
@CodersBrothers no. A common backend is need to share the partial signed transaction between copayers.
